src/Assets.php

```diff
@@ -117,6 +117,7 @@ public static function get_wc_block_data( $settings ) {
 				'enableReviewRating' => 'yes' === get_option( 'woocommerce_enable_review_rating' ),
 				'productCount'       => array_sum( (array) $product_counts ),
 				'attributes'         => wc_get_attribute_taxonomies(),
+				'wcBlocksAssetUrl'   => plugins_url( 'assets/', __DIR__ ),
```

Comment:
I think this URL will be reliable both in the standalone plugin and when merged into core, but I would appreciate a couple of eyes more to confirm it.

Reply:
Could we embed any SVG instead?

Reply:
I'm generally in favor of embedding svg's too.  But a caution:  embedded SVG's will be compiled with the javascript which also increases the size of the javascript bundle (and anything bundling `ErrorBoundary` because it _might_ be used).  Advantages of the image src approach: 

- the image is only downloaded over the wire if it's actually used (shows up in the dom).  
- multiple blocks bundled with ErrorBoundary will not have the SVG adding to their bytes and will share the same image (one download).

Reply:
Agree with @nerrad here. This SVG is much bigger than all other ones we are using in other parts of the codebase, that's why I opted for using an `<img>` tag instead of directly embedding it.